notebooks/fairness-dashboard-loan-allocation.ipynb

```diff
@@ -1,332 +1,1366 @@
-{
```

Comment:
Is this supposed to have changed?

Reply:
Yes, reverting to no net changes for the entirety of the PR